en/docs/administer/using-asymmetric-encryption.md

```diff
@@ -1,196 +1,95 @@
-sensitive information and establishing trust between your server and
-outside parties that connect to the server. The usage of keys and
-certificates contained in a keystore are explained below.
-
-**Keys:** According to public-key cryptography, the concept of a key
-pair (public key and the corresponding private key) is used for
-protecting sensitive information and for authenticating the identity of
-external parties that communicate with your server. For example, the
-information that is encrypted in your server using the public key can
-only be decrypted using the corresponding private key. Therefore, if any
-party wants to decrypt this encrypted data, they should have the
-corresponding private key, which is usually kept as a secret (not
-publicly shared).
-
-**Digital certificate:** When there is a key pair, it is also necessary
-to have a digital certificate to verify the identity of the keys.
-Typically, the public key of a key pair is embedded in this digital
-certificate, which also contains additional information such as the
-owner, validity, etc. of the keys. For example, if an external party
-wants to verify the integrity of data or validate the identity of the
-signer (by validating the digital signature), it is necessary for them
-to have this digital certificate of the signer.
-
-**Trusted certificates and certificate signing authorities:** To
-establish trust, the digital certificate containing the public key
-should be signed by a trusted certificate signing authority (CA). You
-can generate self-signed certificates for the public key (thereby
-creating your own certifying authority), or you can get the certificates
-signed by the digital certificate of an external CA. When the
-certificate is signed by a reputed CA, all the parties that trust this
-CA will also trust the certificates signed by them. To establish maximum
-trust, it is important to have a root CA directly sign your public key
-certificate, or else, you can have an intermediate CA certificate (which
-is already signed by a root CA) sign your certificate. Therefore, in the
-later case, there can be a chain of CAs involved in signing your public
-key certificate. However, note that both types of public key
-certificates (self-signed or CA-signed) can be effectively used
-depending on the sensitivity of the information that is protected by the
-keys.
-
-In summary, each trust chain entry in a keystore contains the
-following:
+WSO2 Identity Server uses asymmetric encryption by default for the authentication and protection of data. In asymmetric encryption, keystores (with private keys and public key certificates) and
+truststores (with only public key certificates) are created and stored. It is possible to have multiple keystores so that the keys used for different use cases are kept unique. The following topics explain more details on keystores and truststores, and how they are used in WSO2 Identity Server.
```

Comment:
`In asymmetric encryption, keystores (with private keys and public key certificates) and
truststores (with only public key certificates) are created and stored.`.

Let's remove this sentence. It doesn't add anything valuable to the information.

Better if we can add a brief intro on what asymmetric encryption is

Reply:
Fixed